src/main/steps/common/task-list/components/notification-banner/notification-banner.macro.njk

```diff
@@ -31,7 +31,7 @@
     {{ govukNotificationBanner({
       titleText: notification.title,
       html: html
-    }) 
+    })
   }}
   {% endfor %}
-{% endmacro %}
\ No newline at end of file
+{% endmacro %}
```

Comment:
why changes are there in src/main/steps/common/task-list/components/notification-banner/notification-banner.macro.njk?

Reply:
Extra space removed shouldn't be an issue